src/acquisition/covidcast/database.py

```diff
@@ -273,13 +371,20 @@ def delete_batch(self, cc_deletions):
     - source
     - time_type
     """
```

Comment:
should we have a check here to make sure there's nothing in the load table? seems like a very very edge case, but could result in inconsistent deletions.

Reply:
+1

Reply:
good idea.  i will remove the `OR REPLACE` in the temp table creation statement, which will cause it to error out if it already exists (it should NOT exist before this method is run, regardless of whether it has anything in it).